config/exporters/monitoring/prometheus/base/prometheus-exporter-service.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: exporter-sa
+  namespace: appstudio-exporter-service-prometheus
```

Comment:
I would suggest `appstudio-grafana-datasource-exporter`

Reply:
Updated to `appstudio-grafana-datasource-exporter`